### PR TITLE
docs: Modify the introduction of the homepage Readme

### PR DESCRIPTION
**Pull request by zhenlei520**

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+* Configuration: 配置中心
+* CQRS
+* DDD
+* Dispatcher
+* Isolation: 支持物理隔离、逻辑隔离
+* MinimalAPI: 支持类似Controller的API分类聚合
+* UoW: 工作单元
+* Storage: 云存储
+
+
```
Comment (doddgu) [inline README.zh-CN.md:26]: roadmap

**Review by doddgu — CHANGES_REQUESTED** [review]

File: README.zh-CN.md (hunk 1 above)
Comment (zhenlei520) [reply README.zh-CN.md:26]: ok.  I will complete the roadmap later

Comment (doddgu) [reply README.zh-CN.md:26]: Just need add roadmap issue link.

Comment (zhenlei520) [reply README.zh-CN.md:32]: processed

File: README.zh-CN.md [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+## 为什么选择MASA.BuildingBlocks？
+* 支持单体架构，也支持分布式架构
+* 基于.Net主推技术栈，不魔改，学习成本低
+* 提供统一的接口标准，有MASA Contrib的最佳实践，但不局限于[MASA Contrib](https://github.com/masastack/MASA.Contrib/blob/main/README.zh-CN.md)，任何人都可以基于MASA BuildingBlocks实现自己的Contrib，重新定制某些特性
+
+
+
+## 了解MASA.BuildingBlocks
```
Comment (doddgu) [inline README.zh-CN.md:15]: Roadmap

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**